content/es-us/docs/latest/ASS_Tags.md

```diff
@@ -0,0 +1,816 @@
+
+{{<tag-def-box title="Establecer color" id="\c">}}
+\\c&H<i>\<bb>\<gg>\<rr></i>&
+\\1c&H<i>\<bb>\<gg>\<rr></i>&
+\\2c&H<i>\<bb>\<gg>\<rr></i>&
+\\3c&H<i>\<bb>\<gg>\<rr></i>&
+\\4c&H<i>\<bb>\<gg>\<rr></i>&
+{{</tag-def-box>}}
+Establece el color del siguiente texto. La etiqueta `\c` es una abreviatura de `\1c`.
+
+- `\1c` establece el color de relleno principal.
+- `\2c` establece el color de relleno secundario. Esto sólo se usa para pre resaltado en karaoke estándar.
+- `\3c` establece el color del borde.
+- `\4c` establece el color de la sombra.
+
+Los códigos de color se dan en [hexadecimal](http://en.wikipedia.org/wiki/Hexadecimal) en orden Azul Verde Rojo. Tenga en cuenta que este es el orden opuesto a los códigos de color HTML. Los códigos de color siempre deben comenzar con `&H` y terminar con `&`.
+
+Los botones Seleccionar Color de la barra de herramientas ![pick-color-toolbar-buttons](/img/3.2/pick-color-toolbar-buttons.png) pueden ayudar a seleccionar colores e ingresar los códigos de color.
+
+{{<tag-def-box title="Establecer alfa" id="\alpha">}}
+\\alpha&H<i>\<aa></i>
+\\1a&H<i>\<aa></i>
+\\2a&H<i>\<aa></i>
+\\3a&H<i>\<aa></i>
+\\4a&H<i>\<aa></i>
+{{</tag-def-box>}}
+Establece el alfa (transparencia) del texto.
+
+- `\alpha` establece el alfa de todos los componentes a la vez.
+- `\1a` establece el alfa de relleno primario.
+- `\2a` establece el alfa de relleno secundario. Esto sólo se usa para pre resaltado en karaoke estándar.
+- `\3a` establece el borde alfa.
+- `\4a` establece la sombra alfa.
+
+Un alfa de 00 (cero) significa opaco/completamente visible, y un alfa de FF (es decir, 255 en decimal) es completamente transparente/invisible.
+
+{{<example-box>}}
+
+```plaintext
+\alpha&H80&
+```
+
+Fija el alfa de todos los componentes en hexadecimal 80, decimal 128, para que el texto sea 50% transparente en general.
+{{</example-box>}}
+{{<example-box>}}
+
+```plaintext
+\1a&HFF&
+```
+
+Fija el alfa de relleno principal en hexadecimal FF, decimal 255, para que resulte efectivamente invisible dejando solo el borde y la sombra.
+{{</example-box>}}
+
+{{<tag-def-box title="Alineación de línea" id="\an">}}\\an<i>\<pos></i>{{</tag-def-box>}}
+Especifica la alineación de la línea. La alineación define la posición de la línea cuando no hay ni [anulación de posición]({{<relref path="ASS_Tags#\pos">}}) ni [movimiento]({{<relref path="ASS_Tags#\move">}}) en efecto. De lo contrario, especifica el punto de anclaje de posicionamiento y rotación.
+
+La etiqueta `\an` usa valores del "teclado numérico" ("numpad") para _pos_. Es decir, los valores de alineación corresponden a las posiciones de los dígitos en el teclado numérico de cuadrícula:
+
+1. Bajo izquierda
+1. Bajo centro
+1. Bajo derecha
+1. Centro izquierda
+1. Centro
+1. Centro derecha
+1. Arriba izquierda
+1. Arriba centro
+1. Arriba derecha
```

Comment:
All 1s? XD

Reply:
Haha yes; most editors for markdown will take care of automatically updating the list item numbers, but since mine did not it was easier to just use `1` always, since it is rendered correctly regardless.